src/main.tsx

```diff
@@ -375,6 +427,19 @@ class PluginUpdateCheckerSettingsTab extends PluginSettingTab {
                     )
                     .setDynamicTooltip()
             );
+        new Setting(containerEl)
+            .setName('Show Notification on New Update')
+            .setDesc('Show a notification when a new update is available')
```

Comment:
```suggestion
            .setDesc('Show a notification when a new update is available. Useful on mobile since the plugin update icon is less visible.')
```